CHANGELOG.md

```diff
@@ -19,9 +19,9 @@
 - Added serialization/deserialization for `MastForest` (#1370)
 - Updated CI to support `CHANGELOG.md` modification checking and `no changelog` label (#1406)
 - Introduced `MastForestError` to enforce `MastForest` node count invariant (#1394)
-- Added functions to `MastForestBuilder` to allow ensuring of nodes with fewer LOC (#1404)
-- Make `Assembler` single-use (#1409)
-- Remove `ProcedureCache` from the assembler (#1411).
+- Added functions to `MastForest` and `MastForestBuilder` to add and ensure nodes with fewer LOC (#1404, 1412)
```

Comment:
nit: `1412` should be `#1412` (this way it will become a link when copied into [release notes](https://github.com/0xPolygonMiden/miden-vm/releases)).

Reply:
Thanks yea, typo